packages/yew/src/virtual_dom/listeners.rs

```diff
@@ -0,0 +1,841 @@
+use std::{
+    cell::RefCell,
+    collections::{HashMap, HashSet},
+    ops::Deref,
+    rc::Rc,
+};
+use wasm_bindgen::{prelude::*, JsCast};
+use web_sys::{Element, Event};
+
+thread_local! {
+    /// Global event listener registry
+    static REGISTRY: RefCell<Registry> = Default::default();
+
+    /// Key used to store listener id on element
+    static LISTENER_ID_PROP: wasm_bindgen::JsValue = "__yew_listener_id".into();
+
+    /// Cached reference to the document body
+    static BODY: web_sys::HtmlElement = crate::utils::document().body().unwrap();
```

Comment:
If yew is mounted on a specific element which is not body, would this still be correct to do?

Reply:
Any events from apps mounted not as the body would still bubble up to the body. All apps share the save event registry.

Reply:
Yes, but I was more curious about if something outside of yew would start triggering yew event handlers because their events are bubbling up to the `body` element?

Reply:
That can only happen, if you have non-yew elements inside the yew VDOM tree, in which case it matches the current behavior. In all other cases it will travel up from the event target and find no yew handlers to execute.

Reply:
Awesome :) thank you :D